src/main/resources/entity-types/inventory/simple_instance.json5

```diff
@@ -947,28 +947,6 @@
       visibleByDefault: false,
       valueGetter: ":sourceAlias.jsonb->>'statusId'",
     },
-    {
```

Comment:
please either update the `V0POCMigration.java` (ln 96) or add a new migration script to change `instance.status` to the new field.